developer-guide/plugin/plugin-editor.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Plugin Editor
+sidebar_position: 2
+---
+
+You can also try our new [Plugin Editor](https://app.reearth.io/plugin-editor) to aid you in developing your own plugin:
+
+<iframe width="100%" height="640" src="https://app.reearth.io/plugin-editor" title="Re:Earth Plugin Editor"></iframe>
+
+## Introduction 
+
+How to make the process of developing plugins simple has always been a problem that our team wants to solve. The main problem is that there is no real-time feedback during the development process, so developers have to repeatedly install and update the plugin. This tool can help us a lot.
+
+Modify your plugin in the code box on the right, and you can see the effect in real time in the left window. This is very helpful for debugging interface and style.
+
+## Still missing
+
+- Calls to the Re:Earth api are not yet supported
+- Still does not support modifying the yml file to customize the interface with the system (right panel property)
+- Code highlighting is still not supported

```

Comment:
If you mean basic syntax highlighting, that is currently supported (for Typescript). Or do you mean something else by this?

Reply:
I mean for the html highlight ...